analysers/analyser_osmosis_parking_highway.py

```diff
@@ -46,7 +48,8 @@
   pr.tags->'park_ride' != 'no'
 FROM
   ways AS pr
-  LEFT JOIN park_highway as highway ON
+  LEFT JOIN park_highway AS highway ON
+    pr.linestring && highway.linestring AND
```

Comment:
This change is not required. `ST_Intersects` already use the index.

https://postgis.net/docs/ST_Intersects.html